DATA/production/qc-async/zdc.json

```diff
@@ -0,0 +1,56 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "alio2-cr1-hv-qcdb1.cern.ch:8083",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul": {
+        "url": "http://ali-consul.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "http://o2-ccdb.internal"
+      }
+    },
+    "tasks": {
+      "ZDCRec": {
+        "active": "true",
+        "className": "o2::quality_control_modules::zdc::ZDCRecDataTask",
+        "moduleName": "QcZDC",
+        "detectorName": "ZDC",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource": {
```

Comment:
Please change the sampling mode to `direct` here.

Reply:
done